charactr_api/sdk/tts_stream_simplex.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import websocket
+import threading
+
+from .data_objects import Credentials, TTSMsgType
+from .config import WS_API_URL
+from typing import Callable
+
+
+class TTSStreamSimplex:
+    def __init__(
+        self,
+        credentials: Credentials,
+        voice_id: int,
+        text: str,
+        blocking: bool = True,
```

Comment:
is there a case, where we would want this to be false? Not sure we need this parametrized.

Reply:
I guess someone might not want this function to block execution and handle it themselves by waiting on the `on_closed` callback.